tests/ffis.test.ts

```diff
@@ -1,122 +1,148 @@
+    expect(
+      INCHIAPI.CheckINCHI(
+        "InChI=1S/C55H73N4O5.Mg/c1-13-39-35(8)42-28-44-37(10)41(24-25-48(60)64-27-26-34(7)23-17-22-33(6)21-16-20-32(5)19-15-18-31(3)4)52(58-44)50-51(55(62)63-12)54(61)49-38(11)45(59-53(49)50)30-47-40(14-2)36(9)43(57-47)29-46(39)56-42;/h13,26,28-33,37,41,51H,1,14-25,27H2,2-12H3,(H-,56,57,58,59,61);/q-1;+2/p-1/b34-26+;/t32-,33-,37+,41+,51-;/m1./s1",
+        0
+      )
+    ).toBe(0); // Standard InChI
+  });
 
-/**
- * Returns string length.
- */
-strict.equal(INCHIAPI.GetStringLength("VNWKTOKETHGBQD-UHFFFAOYSA-N"), 27);
-strict.equal(INCHIAPI.GetStringLength("4444"), 4);
+  test("Check it returns string length", () => {
+    /**
+     * Returns string length.
+     */
+    expect(INCHIAPI.GetStringLength("VNWKTOKETHGBQD-UHFFFAOYSA-N")).toBe(27);
+    expect(INCHIAPI.GetStringLength("4444")).toBe(4);
+  });
 
-/**
- * Test MakeINCHIFromMolfileText
- */
-const output = refNAPI.alloc(inchi_Output, {
-  szInChI: "",
-  szAuxInfo: "",
-  szMessage: "",
-  szLog: "",
-});
-strict.equal(INCHIAPI.MakeINCHIFromMolfileText("", "-SNON -ChiralFlagOFF", output), 0);
-strict.equal(output.deref().szInChI, "");
-
-const output2 = new inchi_Output({
-  szInChI: "",
-  szAuxInfo: "",
-  szMessage: "",
-  szLog: "",
-});
-// Methane mol file | Ref: http://www.cheminfo.org/Chemistry/Generate_molfiles/index.html
-strict.equal(
-  INCHIAPI.MakeINCHIFromMolfileText(
-    `
-Actelion Java MolfileCreator 1.0
+  test("Test MakeINCHIFromMolfileText", () => {
+    /**
+     * Test MakeINCHIFromMolfileText
+     */
+    const output = refNAPI.alloc(inchi_Output, {
+      szInChI: "",
+      szAuxInfo: "",
+      szMessage: "",
+      szLog: "",
+    });
+    expect(INCHIAPI.MakeINCHIFromMolfileText("", "-SNON -ChiralFlagOFF", output)).toBe(0);
+    expect(output.deref().szInChI).toBe("");
 
-  1  0  0  0  0  0  0  0  0  0999 V2000
-    0.0000   -0.0000   -0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
-M  END
-`,
-    "",
-    output2.ref()
-  ),
-  0
-);
-strict.equal(output2.szInChI, "InChI=1S/CH4/h1H4");
+    const output2 = inchi_Output({
+      szInChI: "1S/C5H10/c1-2-4-5-3-1/h1-5H2",
+      szAuxInfo: "",
+      szMessage: "",
+      szLog: "",
+    });
+    // Methane mol file | Ref: http://www.cheminfo.org/Chemistry/Generate_molfiles/index.html
+    expect(
+      INCHIAPI.MakeINCHIFromMolfileText(
+        `C1CCCC1
+        JME 2016-02-24 Sat Mar 05 09:08:54 GMT+100 2016
+        
+          5  5  0  0  0  0  0  0  0  0999 V2000
+            1.8326    0.0000    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+            2.2652    1.3315    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+            1.1326    2.1544    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+            0.0000    1.3315    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+            0.4326    0.0000    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+          1  2  1  0  0  0  0
+          2  3  1  0  0  0  0
+          3  4  1  0  0  0  0
```

Comment:
Why were the mol files changed?

Reply:
@maneetgoyal sir, previous one was failing. I think data was wrong